### DISCUSSION TITLE
Timeline for Android App?

### DISCUSSION THREAD

**Discussion by bparvez** [2fast-team/2fast#100] [General]
Hi, I was just wondering if there is a timeline for an android app release?
It would be great to have a way to sync the 2fast database with the Android App so we can ensure that we have our 2FA codes when away from the desk.

**Comment by jp-weber** [comment]
Hello, currently the plan is to release the Android and iOS version by the end of January. The biggest work in progress at the moment is adding accounts using the camera and native optimization for the respective platform.
![photo_2023-08-25_21-05-34](https://github.com/2fast-team/2fast/assets/10946850/eb48aaf2-98fd-49f4-8145-a79eb7126c9f)

**Reply by Olamite** [reply]
Fingers crossed 🤞 this January ending 🚀

**Reply by katastrophe92** [reply]
@jp-weber  Hello 2Fast Team and community! The App looks nice. I would have no problem if I need to copy the QR-Code to add a account. Is there a new timeline for the android app release? Would you release it for Beta testing? Thanks!

**Reply by jp-weber** [reply]
@katastrophe92
I could compile a preview version next week.

**Reply by katastrophe92** [reply]
> @katastrophe92
> I could compile a preview version next week.

That would be awesome

**Reply by jp-weber** [reply]
I have compiled a test version. However, there are some limitations, see release notes:
https://github.com/2fast-team/2fast/releases/tag/Android-preview-1
